controllers/storageclaim_controller.go

```diff
@@ -398,15 +404,27 @@ func (r *StorageClaimReconciler) reconcilePhases() (reconcile.Result, error) {
 				// storageclaim is clusterscoped resources using its
 				// hash as the clusterID
 				data["clusterID"] = r.storageClaimHash
-				if resource.Name == "cephfs" {
-					volumeSnapshotClass = r.getCephFSVolumeSnapshotClass(data)
-				} else if resource.Name == "ceph-rbd" {
-					volumeSnapshotClass = r.getCephRBDVolumeSnapshotClass(data)
-				}
+				labels["ramendr.openshift.io/storageID"] = storageID
+				volumeSnapshotClass = r.getCephDriverVolumeSnapshotClass(resource.Name, data)
+				utils.AddLabels(volumeSnapshotClass, labels)
```

Comment:
If we are only concerned with a single-label why do we need the labels map var in the first place
```suggestion
				volumeSnapshotClass = r.getCephDriverVolumeSnapshotClass(resource.Name, data)
				utils.AddLabel(volumeSnapshotClass, "ramendr.openshift.io/storageID", storageID)

Reply:
For now. Who knows what will be added next